view/theme/frio/css/style.css

```diff
@@ -1620,6 +1620,7 @@ textarea#profile-jot-text:focus + #preview_profile-jot-text {
 .panel .panel-body {
     padding: 15px;
     font-size: 14px;
+	word-wrap: break-word;
```

Comment:
Wrong indenting compared to the rest of the file (I'm sorry).

Reply:
So 100% wrong code haha